f2/src/App.js

```diff
@@ -11,69 +11,74 @@ import { ThemeProvider, Flex, Link, CSSReset } from '@chakra-ui/core'
 import canada from './theme/canada'
 import { SkipLink } from './components/skip-link'
 import { StateProvider, initialState, reducer } from './utils/state'
+import { IntlProvider } from 'react-intl'
 
 const App = () => {
   const { i18n } = useLingui()
 
   return (
-    <ThemeProvider theme={canada}>
-      <StateProvider initialState={initialState} reducer={reducer}>
-        <CSSReset />
-        <Global
-          styles={css`
-            @import url('https://fonts.googleapis.com/css?family=Noto+Sans:400,400i,700,700i&display=swap');
-          `}
-        />
-        <Flex direction="column" minHeight="100vh">
-          <header>
-            <SkipLink invisible href="#main">
-              <Trans id="SkipLink.text" />
-            </SkipLink>
-            <WarningBanner>
-              <Trans id="banner.warning" />
-            </WarningBanner>
-            <PhaseBanner phase={<Trans id="banner.phase" />}>
-              <Trans id="banner.phaseText" />
-            </PhaseBanner>
-            <TopBanner lang={i18n.locale} bg="black" />
-          </header>
+    <IntlProvider locale={i18n.locale}>
```

Comment:
Are all the changes in this file just indentation because you added this provider?

Reply:
Imndentation does look wonky. I'll pretify it